java/ql/src/experimental/Security/CWE/CWE-089/MyBatisAnnotationSqlInjectionLib.qll

```diff
@@ -0,0 +1,128 @@
+/**
+ * Provides classes for SQL injection detection in MyBatis annotation.
+ */
+
+import java
+import MyBatisCommonLib
+import semmle.code.xml.MyBatisMapperXML
```

Comment:
Given this query isn't related to XML, should this just be renamed `MyBatisMapper`?

Reply:
Here is the modeling of the XML file of MyBatis Mapper. I don't think it needs to be changed. Do you think it must be changed?